ci/docs-builder.py

```diff
@@ -67,16 +76,8 @@ def main():
 
     builder.ensure_dir(WORKING_DIR, clean=True)
 
-    print "Getting git repos"
-    for component in get_components(configuration):
-        #clone the repos
-        branch_name = component['git_branch']
-        print "Cloning from github: %s" % component.get('git_url')
-        print "Switching to branch %s" % branch_name
-        clone_command = ['git', 'clone', component.get('git_url'), '--branch', branch_name]
-        exit_code = subprocess.call(clone_command, cwd=WORKING_DIR)
-        if exit_code != 0:
```

Comment:
check_call does this. If the update-version scripts fails, this script will fail.